config/pinpoint_overrides.yml

```diff
@@ -0,0 +1,160 @@
+AE:
+  supports_voice_unconfirmed: false
+  supports_sms: false
+  supports_voice: true
```

Comment:
currently the docs do not include the UAE for voice... should we remove this?
https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-voice-countries.html

Reply:
> currently the docs do not include the UAE for voice... should we remove this?
> https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-voice-countries.html

Seems reasonable to me if it's not listed as supported? I'd also wondered about some of the other removals like Angola, Algeria, Bolivia, etc, but assumed those were expected since they're not supported by AWS.

Reply:
Removed all overrides to true, since if Pinpoint doesn't support... we can't support in ad03b651066062e3482971f4812161c334ecf04a